/**:
  ros__parameters:
    converged_steer_rad: 0.15
    resampling_ds: 0.07
    lookahead_distance: 0.0
    closest_thr_dist: 3.0
    closest_thr_ang: 0.785
    trajectory_input_points: 10
    model_path: "model_fused_data(1).onnx"
    precision: "fp32"